### ISSUE TITLE
Use IMVDb for music videos?

### ISSUE THREAD

**Issue by z1lc** [z1lc/zdone#62]
https://imvdb.com/

**Comment by wclausen** [comment]
i like the concept of including a link to the music videos in the cards. would be cool to get more into music videos. right now, there's a discoverability problem for me where I don't even know if a given song has a music video and don't have the habit of listening to music on youtube

**Comment by z1lc** [comment]
Yeah this general idea would require us to find a mapping between Spotify artist <> IMVDb artist, and then perhaps further Spotify song <> IMVDb song. The first one should be pretty easy, the second might be a little tougher. Then we could add the link to the same cards that we're already reviewing in Spotify+Anki.

**Comment by z1lc** [comment]
Seems like their `entity` endpoint doesn't really work. Might be easiest just to use artist name + song name and hit the `search` endpoint, doing some sanity checking on the top result to see if it actually matches.

https://imvdb.com/api/v1/search/videos?q=50%20Cent%20Just%20A%20Lil%20Bit

First result is indeed the one we're looking for, and manually testing it seems like this is consistently the case.
